### PR TITLE
User Management - list request performance

### PR DESCRIPTION
**Pull request by Sebastian-Haehnlein**
## PR Type

[x] Refactoring (no functional changes, no API changes)

## What Is the New Behavior?

no unnecessary requests on user list page.

### CODE REVIEW

File: projects/organization-management/src/app/models/b2b-user/b2b-user.model.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { User } from 'ish-core/models/user/user.model';
+
+export interface B2bUser extends User {
```
Comment (Sebastian-Haehnlein) [inline projects/organization-management/src/app/models/b2b-user/b2b-user.model.ts:3]: name ok?

File: projects/organization-management/src/app/models/b2b-user/b2b-user.model.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  name?: string; // list call only
```
Comment (Sebastian-Haehnlein) [inline projects/organization-management/src/app/models/b2b-user/b2b-user.model.ts:4]: i will do a change request to get firstName, lastName instead. (more compatible)

File: projects/organization-management/src/app/models/b2b-user/b2b-user.mapper.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import { B2bUser } from './b2b-user.model';
+
+export class B2bUserMapper {
+  static fromData(user: UserData): B2bUser {
+    // is used for business users
+    return UserMapper.fromData(user);
+  }
+
+  static fromListData(data: { elements: { name: string; login: string }[] }): B2bUser[] {
```
Comment (dhhyi) [inline projects/organization-management/src/app/models/b2b-user/b2b-user.mapper.ts:12]: test is missing

File: projects/organization-management/src/app/models/b2b-user/b2b-user.mapper.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    // is used for business users
```
[the remark this answers is not in the document]
Comment (Sebastian-Haehnlein) [reply projects/organization-management/src/app/models/b2b-user/b2b-user.mapper.ts:13]: removed

**Pull request merged**